File: backend/src/main/java/ru/yakimov/aviato/model/Person.java
package ru.yakimov.aviato.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import ru.yakimov.aviato.constants.Role;



@Data

@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Person {

    @Id
    private Integer id;
    private String username;
    private String password;
    private String firstName;
    private String middleName;
    private String lastName;
    private Role role;

}

File: frontend/aviato/src/app/model/person.ts

export enum Role {
  GUEST = "GUEST",
  USER = "USER",
  ADMIN = "ADMIN"
}

export interface Person {
  id: number,
  username: string,
  role: Role,
  first_name: string,
  middle_name: string,
  last_name: string
}
